sdk/simulation/uinst/td_mngr.h

```diff
@@ -56,6 +56,15 @@ typedef struct _tcs_sim_t
 #define GET_TCS_PTR(xbp) (tcs_t *)(*(uintptr_t *)((size_t)(xbp) + 2 * sizeof(uintptr_t)))
 #endif
 
+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
It may not be proper to add the macro to this file as it is only for td/tcs related.  How about adding them to sdk/simulation/assembly/lowlib.h?

Reply:
Moved.